pkg/infinispan/grafana/dashboard_controller.go

```diff
@@ -0,0 +1,43 @@
+package grafana
```

Comment:
for me `dashboard_controller.go` a bit confusing name as it's not classic controller with Watchers and Reconciler
Probably we need a new name for it, but if we will need to control deletion of this CR, we need to implement controller part for it and keep this name the same